package.json

```diff
@@ -11,30 +11,30 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^15.1.3",
-    "@angular/common": "^15.1.3",
-    "@angular/compiler": "^15.1.3",
-    "@angular/core": "^15.1.3",
-    "@angular/forms": "^15.1.3",
-    "@angular/platform-browser": "^15.1.3",
-    "@angular/platform-browser-dynamic": "^15.1.3",
-    "@angular/router": "^15.1.3",
+    "@angular/animations": "^16.0.0",
+    "@angular/common": "^16.0.0",
+    "@angular/compiler": "^16.0.0",
+    "@angular/core": "^16.0.0",
+    "@angular/forms": "^16.0.0",
+    "@angular/platform-browser": "^16.0.0",
+    "@angular/platform-browser-dynamic": "^16.0.0",
+    "@angular/router": "^16.0.0",
     "rxjs": "~7.5.0",
     "tslib": "^2.3.0",
-    "zone.js": "~0.11.4"
+    "zone.js": "~0.13.0"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^15.1.4",
-    "@angular/cli": "~15.1.4",
-    "@angular/compiler-cli": "^15.1.3",
+    "@angular-devkit/build-angular": "^16.0.0",
+    "@angular/cli": "~16.0.0",
+    "@angular/compiler-cli": "^16.0.0",
     "@types/jasmine": "~4.0.0",
     "jasmine-core": "~4.1.0",
     "karma": "~6.3.0",
     "karma-chrome-launcher": "~3.1.0",
     "karma-coverage": "~2.2.0",
     "karma-jasmine": "~5.0.0",
     "karma-jasmine-html-reporter": "~1.7.0",
-    "ng-packagr": "^15.1.1",
+    "ng-packagr": "^16.0.0",
     "typescript": "~4.9.5"
```

Comment:
Should it not have TypeScript v5?

Reply:
the minimal typescript version for Angular 16 is 4.9, thats why the update schematics don't change it.
But we can upgrade to v5